src/gitlab.js

```diff
@@ -10,6 +10,9 @@ class GitLab {
     this.per_page = args.per_page || 100;
     this.projects = null;
     this.avatar_cache = args.avatar_cache;
+    this.cur_proj_name = null;
+    this.branchs = null;
```

Comment:
Plural form of `branch` is not `branchs` but `branches`

Reply:
sorry, fixed this